### ISSUE TITLE
Register as Bower package

### ISSUE THREAD

**Issue by pbojinov** [pbojinov/nodeifications#12]
Figure out how the this solution can be a standalone npm module or bower package. The only tricky part is resolving the dependency of the HTML from within the js.

**Comment by cirocosta** [comment]
It seems more like a bower package than a npm module. What do you think?

**Comment by pbojinov** [comment]
@cirocosta you are right. Since it will be used by a "client-side" app within a node app.
